backend/src/main/resources/XSLT/html/literature.xslt

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+				xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+				xmlns:akn="http://docs.oasis-open.org/legaldocml/ns/akn/3.0"
+				xmlns:ris="http://ldml.neuris.de/literature/metadata/"
+				exclude-result-prefixes="akn ris">
+
+	<xsl:output method="html"/>
```

Comment:
Should we also add the encoding and indentation?

`<xsl:output method="html" encoding="UTF-8" indent="yes"/>`